openbas-front/src/admin/components/components/teams/AddTeams.tsx

```diff
@@ -125,90 +130,25 @@ const AddTeams: React.FC<Props> = ({ addedTeamIds, onAddTeams }) => {
-                    || addedTeamIds.includes(team.team_id);
-                  return (
-                    <ListItem
-                      key={team.team_id}
-                      disabled={disabled}
-                      button
-                      divider
-                      dense
-                      onClick={() => setTeamIds([...teamIds, team.team_id])}
-                    >
-                      <ListItemIcon>
-                        <GroupsOutlined />
-                      </ListItemIcon>
-                      <ListItemText
-                        primary={team.team_name}
-                        secondary={team.organization_name}
-                      />
-                      <ItemTags variant="reduced-view" tags={team.team_tags} />
-                    </ListItem>
-                  );
-                })}
-                <CreateTeam
-                  inline
-                  onCreate={(team) => setTeamIds([...teamIds, team.team_id])}
-                />
-              </List>
-            </Grid>
-            <Grid item xs={4}>
-              <Box className={classes.box}>
-                {teamIds.map((teamId) => {
-                  const team = teamsMap[teamId];
-                  if (!team) {
-                    return null; // Skip rendering if team is undefined
-                  }
-                  const teamGravatar = R.propOr('-', 'team_gravatar', team);
-                  return (
-                    <Chip
-                      key={teamId}
-                      onDelete={() => {
-                        setTeamIds((prevIds) => prevIds.filter((id) => id !== teamId));
-                      }}
-                      label={truncate(team.team_name, 22)}
-                      avatar={<Avatar src={teamGravatar} sx={{ height: '32px', width: '32px' }} />}
-                      classes={{ root: classes.chip }}
-                    />
-                  );
-                })}
-              </Box>
-            </Grid>
-          </Grid>
+          <Box sx={{ marginTop: 2 }}>
+            <SelectList
+              values={teams}
+              selectedValues={teamValues}
+              elements={elements}
+              prefix="team"
+              onSelect={addTeam}
+              onDelete={removeTeam}
+              paginationComponent={paginationComponent}
+              buttonComponent={<CreateTeam
+                inline
+                onCreate={(team) => setTeamValues([...teamValues, team])}
+                               />}
+            />
+          </Box>
         </DialogContent>
         <DialogActions>
-          <Button onClick={() => {
-            setOpen(false);
-            setKeyword('');
-            setTeamIds([]);
-          }}
-          >{t('Cancel')}</Button>
-          <Button color="secondary" onClick={submitAddTeams}>
+          <Button onClick={handleClose}>{t('Cancel')}</Button>
+          <Button color="secondary" onClick={handleSubmit}>
             {t('Add')}
```

Comment:
The component's behavior has changed: we can now add or remove teams. It might be more appropriate to update the label to 'Update', what do you think ? 
And also to change the DialogTitle ?

Reply:
Well seen